tests/test_rewriter.py

```diff
@@ -76,17 +80,46 @@ def tti_operator(dse=False):
     return handle
 
 
-def test_tti_rewrite_basic():
-    output1 = tti_operator(dse=None).apply()
-    output2 = tti_operator(dse='basic').apply()
+@pytest.fixture(scope="session")
+def tti_nodse():
+    # FIXME: note that np.copy is necessary because of the broken caching system
```

Comment:
Forgot to try this. I can handle this in the next PR though (as I might have to change that bit of code anyway).